apps/server/src/modules/oauth-provider/service/oauth-provider.login-flow.service.spec.ts

```diff
@@ -68,12 +96,7 @@ describe('OauthProviderLoginFlowService', () => {
 				skip: true,
 				subject: 'subject',
 			} as ProviderLoginResponse;
-			const ltiToolDoMock: LtiToolDO = {
-				id: 'toolId',
-				name: 'name',
-				oAuthClientId: 'oAuthClientId',
-				isLocal: true,
-			};
+			ltiToolDoMock = { ...ltiToolDoMock, id: 'toolId', name: 'name', oAuthClientId: 'oAuthClientId' };
```

Comment:
id: 'toolId' is already set in the before each part of ltiToolDoMock

Reply:
Done